### ISSUE TITLE
Unable to Disable Tailwind CSS

### ISSUE THREAD

**Issue by kumaraguru1735** [reflex-dev/reflex#2361]
**Describe the bug**
According to docs to disable tailwind we need to use
```config = rx.Config(app_name="app", tailwind=None)```

**Output While disable**
```
    app.compile()
  File "e:\OneCloud\Portal\.venv\lib\site-packages\reflex\app.py", line 750, in compile
    compile_results.append(future.result())
  File "C:\Users\PRO\AppData\Local\Programs\Python\Python39\lib\concurrent\futures\_base.py", line 439, in result
    return self.__get_result()
  File "C:\Users\PRO\AppData\Local\Programs\Python\Python39\lib\concurrent\futures\_base.py", line 391, in __get_result
    raise self._exception
  File "C:\Users\PRO\AppData\Local\Programs\Python\Python39\lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "e:\OneCloud\Portal\.venv\lib\site-packages\reflex\compiler\compiler.py", line 431, in compile_tailwind
    code = _compile_tailwind(config)
  File "e:\OneCloud\Portal\.venv\lib\site-packages\reflex\compiler\compiler.py", line 279, in _compile_tailwind
    return templates.TAILWIND_CONFIG.render(
TypeError: jinja2.environment.Template.render() argument after ** must be a mapping, not NoneType
```

**Specifics (please complete the following information):**
 - Python Version: 3.9.13
 - Reflex Version: 0.3.7
 - OS:  windows10

**Comment by kumaraguru1735** [comment]
well fixed on #2246 commit

**Comment by masenf** [comment]
Fix will be released today or tomorrow.
